common/settings/settings.ts

```diff
@@ -39,6 +39,7 @@ export interface AnkiSettings {
     readonly maxImageHeight: number;
     readonly surroundingSubtitlesCountRadius: number;
     readonly surroundingSubtitlesTimeRadius: number;
+    readonly clickToMineDefaultPlayback: PostMinePlayback;
```

Comment:
I would name this setting something else since it applies even when not clicking to mine.